proxy/http/HttpTransact.cc

```diff
@@ -7142,26 +7142,38 @@ HttpTransact::does_client_request_permit_storing(CacheControlResult *c, HTTPHdr
   return (true);
 }
 
+int
+HttpTransact::get_max_age(State *s, HTTPHdr *response)
```

Comment:
Why is the State object a parameter to this function?

Reply:
Good question, state is needed by TxnDebug. I have kept the two debugging statements (printing s_maxage or max_age depending on which one is defined) because they were used in calculate_document_freshness_limit. If we decide that they are not needed, after all, then we can avoid passing State as a parameter to the function.

Reply:
My general thought it that we should avoid duplicating Debug()'s which are already in the core. It's not intended for debugging "line-by-line" code, but rather, show internal stats, transitions, state machine states etc. as the request is being processed.

So, assuming there's an existing Debug that shows the max-age, we should remove it here.

Reply:
> My general thought it that we should avoid duplicating Debug()'s which are already in the core. It's not intended for debugging "line-by-line" code, but rather, show internal stats, transitions, state machine states etc. as the request is being processed.
> 
> So, assuming there's an existing Debug that shows the max-age, we should remove it here.
I couldn't find any, so I've re-added one single debug statement showing max-age to  `calculate_document_freshness_limit`. Let me know if this looks any better!